docs/products/access/api-reference/config-api/identity-provider-api.md

```diff
@@ -93,13 +94,104 @@ Pragma: no-cache
+| token_url                    | OAUTH        | yes      | Oauth token endpoint.                                                                                                                                                                                                   |
+| profile_url                  | OAUTH        | yes      | OpenID Connect UserInfo endpoint.                                                                                                                                                                                       |
+| user_info_enabled            | OAUTH        | no       | Specify whether CIM's Person API is enabled for this Identity Provider. <br/> Default value: `false`.                                                                                                                   |
+| user_info_endpoint           | OAUTH        | depends  | Identity source URL. The URL of API that provides user's identity. Use `{userId}` placeholder for userId path param. e.g. https://host/api/persons/{userId}/profile  <br/> Required when `user_info_enabled` is `true`. |
+| user_info_username           | OAUTH        | depends  | Identity source username. <br/> Required when `user_info_enabled` is `true`.                                                                                                                                            |
+| user_info_password           | OAUTH        | depends  | Identity source password. <br/> Required when `user_info_enabled` is `true`.                                                                                                                                            |
+
+Example `TULIP` type request:
+
+```http
+POST /api/v1/configuration/idps 
+Host: onegini.example.com
+Content-Type: application/json
+{
+   "id":"tulip-insurcar",
+   "name":"Tulip Insurcar Idp",
+   "type":"TULIP",
+   "client_id":"oauth2CustomerApp",
+   "issuer_uri":"https://insurgroup-edge.iwelcome.com/insurcar/login/oauth2/v1",
+   "end_session_enabled":true,
+   "scopes":"openid",
+   "integrations":[
+      "APP_TO_WEB",
+      "UDH_API"
+   ],
+   "tulip_api_client_id":"accessIntegration",
+   "tulip_api_base_url":"https://insurgroup-edge.iwelcome.com/insurcar",
+   "tulip_api_access_scope":"iwelcome:segment:insurgroup"
+}
+```
+
+Example `OAUTH` type request:
+
+```http
+POST /api/v1/configuration/idps 
+Host: onegini.example.com
+Content-Type: application/json
+{
+  "type": "OAUTH",
+  "id": "oauth-idp",
+  "name": "OAuth Identity Provider",
+  "client_id": "oauth2CustomerApp",
+  "authorization_url": "https://onegini.test.onegini.com/oauth/v1/authorize",
+  "token_url": "https://onegini.test.onegini.com/oauth/v1/token",
+  "profile_url": "https://onegini.test.onegini.com/oauth/v1/userinfo",
+  "scopes": "openid profile email"
```

Comment:
Remove onegini naming, we usually try to just use example.com, at least when looking at our previous documentation

```suggestion
  "type": "OAUTH",
  "id": "oauth-idp",
  "name": "OAuth Identity Provider",
  "client_id": "oauth2CustomerApp",
  "authorization_url": "https://example.com/oauth/v1/authorize",
  "token_url": "https://example.com/oauth/v1/token",
  "profile_url": "https://example.com/oauth/v1/userinfo",
  "scopes": "openid profile email"
```